bucket/terminal-icons.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "0.9.0",
+    "description": "A PowerShell module to show file and folder icons in the terminal",
+    "homepage": "https://github.com/devblackops/Terminal-Icons",
+    "license": "MIT",
+    "notes": [
+        "Use the module by running: 'Import-Module Terminal-Icons'",
+        "Add it to your $PROFILE to make it permanent"
+    ],
```

```suggestion
    "license": "MIT",
```

Don't add usage notes in manifest.

---

Its not a usage note, I just followed the other manifests in this repo ([`zlocation`](https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/zlocation.json), [`powertab`](https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/powertab.json), [`posh-docker`](https://github.com/ScoopInstaller/Extras/blob/59b802f3e6b9b9b52c3c632d449afb48f54f023c/bucket/posh-docker.json)).

---

In fact, it is, and the others' should be cleaned up too.

But okay, keep it now and batch clean job will be done later.